FreeBSD: build is broken after #5298

See https://builds.sr.ht/~thestr4ng3r/job/1550499#task-build
```
In file included from ../librz/debug/p/debug_native.c:68:
../librz/debug/p/native/kfbsd.c:202:6: warning: variable 'showfpu' set but not used [-Wunused-but-set-variable]
  202 |         int showfpu = false;
      |             ^
../librz/debug/p/native/kfbsd.c:328:8: warning: variable 'map_is_shared' set but not used [-Wunused-but-set-variable]
  328 |                 bool map_is_shared = false;
      |                      ^
2 warnings generated.
[1924/2337] Compiling C object subprojects/rizin-shell-parser/libshell_parser.a.p/src_scanner.c.o
[1925/2337] Compiling C object librz/debug/librz_debug.so.0.9.0.p/p_native_bsd_bsd_debug.c.o
[1926/2337] Linking target librz/debug/librz_debug.so.0.9.0
FAILED: librz/debug/librz_debug.so.0.9.0 
clang++  -o librz/debug/librz_debug.so.0.9.0 librz/debug/librz_debug.so.0.9.0.p/ddesc.c.o librz/debug/librz_debug.so.0.9.0.p/debug.c.o librz/debug/librz_debug.so.0.9.0.p/dreg.c.o librz/debug/librz_debug.so.0.9.0.p/desil.c.o librz/debug/librz_debug.so.0.9.0.p/dmap.c.o librz/debug/librz_debug.so.0.9.0.p/pid.c.o librz/debug/librz_debug.so.0.9.0.p/plugin.c.o librz/debug/librz_debug.so.0.9.0.p/dsession.c.o librz/debug/librz_debug.so.0.9.0.p/dsignal.c.o librz/debug/librz_debug.so.0.9.0.p/serialize_debug.c.o librz/debug/librz_debug.so.0.9.0.p/snap.c.o librz/debug/librz_debug.so.0.9.0.p/trace.c.o librz/debug/librz_debug.so.0.9.0.p/p_bfvm.c.o librz/debug/librz_debug.so.0.9.0.p/p_common_windows.c.o librz/debug/librz_debug.so.0.9.0.p/p_common_winkd.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_bf.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_bochs.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_dmp.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_gdb.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_io.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_null.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_winkd.c.o librz/debug/librz_debug.so.0.9.0.p/bp.c.o librz/debug/librz_debug.so.0.9.0.p/bp_io.c.o librz/debug/librz_debug.so.0.9.0.p/bp_traptrace.c.o librz/debug/librz_debug.so.0.9.0.p/bp_watch.c.o librz/debug/librz_debug.so.0.9.0.p/serialize_bp.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_qnx.c.o librz/debug/librz_debug.so.0.9.0.p/p_debug_native.c.o librz/debug/librz_debug.so.0.9.0.p/p_native_procfs.c.o librz/debug/librz_debug.so.0.9.0.p/p_native_bsd_bsd_debug.c.o -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,-soname,librz_debug.so.0.9 '-Wl,-rpath,$ORIGIN/../util:$ORIGIN/../cons:$ORIGIN/../socket:$ORIGIN/../hash:$ORIGIN/../crypto:$ORIGIN/../io:$ORIGIN/../bin:$ORIGIN/../demangler:$ORIGIN/../magic:$ORIGIN/../syscall:$ORIGIN/../type:$ORIGIN/../reg:$ORIGIN/../egg:$ORIGIN/../arch:$ORIGIN/../search:$ORIGIN/../config:$ORIGIN/../flag:$ORIGIN/../diff:$ORIGIN/../il' -Wl,-rpath-link,/home/build/rizin/build/librz/util -Wl,-rpath-link,/home/build/rizin/build/librz/cons -Wl,-rpath-link,/home/build/rizin/build/librz/socket -Wl,-rpath-link,/home/build/rizin/build/librz/hash -Wl,-rpath-link,/home/build/rizin/build/librz/crypto -Wl,-rpath-link,/home/build/rizin/build/librz/io -Wl,-rpath-link,/home/build/rizin/build/librz/bin -Wl,-rpath-link,/home/build/rizin/build/librz/demangler -Wl,-rpath-link,/home/build/rizin/build/librz/magic -Wl,-rpath-link,/home/build/rizin/build/librz/syscall -Wl,-rpath-link,/home/build/rizin/build/librz/type -Wl,-rpath-link,/home/build/rizin/build/librz/reg -Wl,-rpath-link,/home/build/rizin/build/librz/egg -Wl,-rpath-link,/home/build/rizin/build/librz/arch -Wl,-rpath-link,/home/build/rizin/build/librz/search -Wl,-rpath-link,/home/build/rizin/build/librz/config -Wl,-rpath-link,/home/build/rizin/build/librz/flag -Wl,-rpath-link,/home/build/rizin/build/librz/diff -Wl,-rpath-link,/home/build/rizin/build/librz/il -Wl,--start-group subprojects/rzgdb/librzgdb.a librz/util/librz_util.so.0.9.0 librz/cons/librz_cons.so.0.9.0 librz/socket/librz_socket.so.0.9.0 subprojects/rzwinkd/librzwinkd.a librz/hash/librz_hash.so.0.9.0 librz/crypto/librz_crypto.so.0.9.0 librz/io/librz_io.so.0.9.0 librz/bin/librz_bin.so.0.9.0 librz/reg/librz_reg.so.0.9.0 librz/syscall/librz_syscall.so.0.9.0 librz/egg/librz_egg.so.0.9.0 librz/arch/librz_arch.so.0.9.0 librz/type/librz_type.so.0.9.0 subprojects/rzqnx/librzqnx.a /usr/lib/libkvm.so -Wl,--end-group
ld: error: undefined symbol: kinfo_getproc
>>> referenced by bsd_debug.c:157 (../librz/debug/p/native/bsd/bsd_debug.c:157)
>>>               librz/debug/librz_debug.so.0.9.0.p/p_native_bsd_bsd_debug.c.o:(bsd_info)
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/librz/debug/meson.build
+++ b/librz/debug/meson.build
@@ -105,6 +105,9 @@ if has_debugger
     ]
   endif
 
+  if host_machine.system() == 'freebsd'
+    rz_debug_deps += cc.find_library('util', required: true, static: is_static_build)
+  endif
   if ['netbsd', 'openbsd', 'freebsd', 'dragonfly'].contains(host_machine.system())
     rz_debug_deps += cc.find_library('kvm', required: true, static: is_static_build)
     rz_debug_sources += [

